Replicator/tests/data/cbl-4418-inject-error-pull.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/Replicator/DBAccess.cc b/Replicator/DBAccess.cc
+index 9efa7493f..46d53482f 100644
+--- a/Replicator/DBAccess.cc
++++ b/Replicator/DBAccess.cc
+@@ -364,6 +364,12 @@ namespace litecore { namespace repl {
+         
+         Doc result;
+         FLError flErr;
++
++        if (string(doc->docID()) == "doc-001") {
```

Comment:
What the heck is this?

Reply:
This file won't go to the source. It is a patch file. It can help to inject artificial delta error to test the code added by this PR. It's explained in the relevant test.